lib/version_utils.pm

```diff
@@ -312,7 +312,7 @@ sub is_svirt_except_s390x {
 
 sub is_remote_backend {
     # s390x uses only remote repos
-    return check_var('ARCH', 's390x') || check_var('BACKEND', 'svirt') || check_var('BACKEND', 'ipmi') || check_var('BACKEND', 'spvm');
+    return check_var('ARCH', 's390x') || check_var('BACKEND', 'ipmi') || check_var('BACKEND', 'spvm');
```

Comment:
As I've mentioned, this is used in multiple places, where svirt is expected to be remote backend, so this change is no good =(

Reply:
@asdil12 Better create another `is_...` sub which just does not pick `svirt` and use it for the `ttyS0` exclusion.

Reply:
I don't do anything about ttyS0 here, so I will just do the check inline.